app/src/main/java/protect/card_locker/MainActivity.java

```diff
@@ -175,17 +221,33 @@ public void onDestroyActionMode(ActionMode inputMode) {
 
     @Override
     protected void onCreate(Bundle inputSavedInstanceState) {
+        extractIntentFields(getIntent());
         super.onCreate(inputSavedInstanceState);
         SplashScreen.installSplashScreen(this);
-        setTitle(R.string.app_name);
+        if(!mArchiveMode) {
+            setTitle(R.string.app_name);
+            setContentView(R.layout.main_activity);
+        }
+        else{
+            setTitle(R.string.archive);
```

Comment:
In English the action and the list are the same, but that's not necessarily the case for other languages. So we need to split this up in 2 strings.

|  | English | Dutch |
| - | - | - |
| Move to archive | Archive | Archiveer |
| Archived cards | Archive | Archief |

```suggestion
            setTitle(R.string.archiveList);
```